code/datums/actions/directional_attack_toggle.dm

```diff
@@ -0,0 +1,20 @@
+//Toggle directional attacks
+/datum/action/directional_attack_toggle
+	name = "Toggle Directional Attacks"
+	var/static/atom/movable/vis_obj/action/bump_attack_active/active_icon = new
+	var/static/atom/movable/vis_obj/action/bump_attack_inactive/inactive_icon = new
+
```

Comment:
I feel like this, being integral to this component, can be in the directional attack component file. If only to have things be slightly easier to port as it's a single file.

Thoughts, other maintainers?